requirements.txt

```diff
@@ -2,7 +2,6 @@ boto
 django-bootstrap3
 django-debug-toolbar
 django-debug-toolbar-template-timings
-django-extensions
```

Comment:
@danaspiegel I didn't know that it is necessary for shell_plus command, I removed it because the only use we were using (UUIDField) in no more needed, as this field is now embedded with the new django version.

I will get it back.